### ISSUE TITLE
Screen reader optimized popup: close button is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52047]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10,MACOS High Sierra 10.13.5 Beta (17F70a)

**Additional Details:**
Keyboard,MAS2.1.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to File->select New File->Navigate to Status Bar->Click on "Screen Reader Optimized" button.
3. Check the keyboard navigation.
**Actual:**
Close button is not accessible by keyboard and also by using  escape key,the screen does not close. 

**Expected:**
Close button should accessible by keyboard.
Same issue observed in MAC also.
**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
**User Impact:**
The keyboard user will not be able to close the screen.
**MAS Reference**:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
Does this issue occur when all extensions are disabled?: Yes

**Attachment for Reference**
[[Accessibility]A11y_VSCode_ScreenReaderOptimized_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2106227/Accessibility.A11y_VSCode_ScreenReaderOptimized_Keyboard.pptx)
[[Accessibility]A11y_VSCode_ScreenReaderOptimized_Keyboard1.pptx](https://github.com/Microsoft/vscode/files/2106228/Accessibility.A11y_VSCode_ScreenReaderOptimized_Keyboard1.pptx)

**Comment by isidorn** [comment]
Even though it is fine that this popup can not be focused, esc should still close it.
Option 2 is to focus into the popup so the text is read, and allow to navigate via tab to close it.
Feels law hanging -> july

**Comment by isidorn** [comment]
@alexandrudima we need this for a better accessilibity grade. Do you mind if I tackle it with a simple esc command that is enalbed only when the popup is active?

**Comment by alexandrudima** [comment]
Perhaps I should migrate the popup to use of the new dialog service methods.

**Comment by alexandrudima** [comment]
Fixed by using a notification instead of custom UI in https://github.com/Microsoft/vscode/commit/1433e180d1ca25fa2a33b900a17b633123120631
